BluetoothLeChat/app/src/main/java/com/example/bluetoothlechat/scan/DeviceScanFragment.kt

```diff
@@ -68,7 +69,8 @@ class DeviceScanFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         _binding = FragmentDeviceScanBinding.inflate(inflater, container, false)
-
+        val devAddr = getString(R.string.your_device_address)+ ChatServer.getYourDeviceAddress()
```

Comment:
Should be a space before + sign. Might want to run through Ktlint. It picks up all this stuff for you.

Reply:
ack